src/citrine/informatics/executions/sample_design_space_execution.py

```diff
@@ -0,0 +1,101 @@
+from functools import partial
+from typing import Optional, Iterable
+from uuid import UUID
+
+from citrine._rest.asynchronous_object import AsynchronousObject
+from citrine._rest.pageable import Pageable
+from citrine._rest.paginator import Paginator
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine._utils.functions import format_escaped_url
+from citrine.informatics.design_candidate import SampleSearchSpaceResultCandidate
+
+
+class SampleDesignSpaceExecution(
+    Resource["SampleDesignSpaceExecution"], Pageable, AsynchronousObject
+):
+    """The execution of a Sample Design Space task.
+
+    Possible statuses are INPROGRESS, SUCCEEDED, and FAILED.
+    Design executions also have a ``status_description`` field with more information.
+    """
+
+    _paginator: Paginator = Paginator()
+    _collection_key = 'response'
+    _in_progress_statuses = ["INPROGRESS"]
+    _succeeded_statuses = ["SUCCEEDED"]
+    _failed_statuses = ["FAILED"]
+    _session: Optional[Session] = None
+    project_id: Optional[UUID] = None
+    design_space_id: Optional[UUID] = None
+
+    uid: UUID = properties.UUID('id', serializable=False)
+    """:UUID: Unique identifier of the execution"""
+    status = properties.Optional(properties.String(), 'status', serializable=False)
```

Comment:
Are there situations where the status won't be defined on the response payload? Or cases where this object will be built as an input payload?